.github/workflows/PR_validation.yml

```diff
@@ -67,11 +72,26 @@ jobs:
             dllib:
               - 'scala/dllib/**'
               - '.github/actions/dllib-scala-ut-action/action.yml'
+            ppml:
+            - 'scala/ppml/**'
+            - '.github/actions/ppml-scala-ut-action/action.yml'
+            - '.github/workflows/PR_validation.yml'
             orca:
               - 'python/orca/**'
               - '.github/actions/orca-ray-ctx-example-action/action.yml'
-
-  
+              - '.github/actions/orca/orca-python-exampletest-action/pr-test/action.yml'
+              - '.github/actions/orca/orca-python-exampletest-ray-action/pr-test/action.yml'
+              - '.github/actions/orca/orca-python-exampletest-horovod-pytorch-py37-spark3-action/pr-test/action.yml'
+              - '.github/actions/orca/orca-python-exampletest-horovod-tf2-py37-spark3-action/pr-test/action.yml'
+              - '.github/actions/orca/orca-jep-exampletest-action/pr-test/action.yml'
+              - '.github/actions/orca/orca-python-py37-spark3-action/pr-test/action.yml'
+              - '.github/actions/orca/orca-python-ray-py37-spark3-action/pr-test/action.yml'
+              - '.github/actions/orca/orca-python-horovod-tf2-py37-spark3-action/pr-test/action.yml'
+              - '.github/actions/orca/orca-python-horovod-pytorch-py37-spark3-action/pr-test/action.yml'
+              - '.github/actions/orca/orca-jep-python-py37-spark2-action/pr-test/action.yml'
+              - '.github/actions/orca/orca-rayonspark-tf2-action/pr-test/action.yml'
```

Comment:
This can be refined to one line:
`- '.github/actions/orca/**'`

Reply:
> This can be refined to one line: `- '.github/actions/orca/**'`

update